### DISCUSSION TITLE
Offboarding OneDrive

### DISCUSSION THREAD

**Discussion by citjezh** [KelvinTegelaar/CIPP#1159] [Ideas]
Hi All, 

The offboarding is great and makes things so much easier. The one issue we are seeing is that now more and more users are using OneDrive there is often a small to large amount of data that the user has, whilst there is the option to share with another person, this means we need to keep the license and then go back and remove the license at a later date. 

I am suggesting two options 
1. Migrate all Data into another users OD under a folder 
2. Migrate into a sharepoint site (we create a User Archive) under that users name, with the option of sharing out that one folder to another user (nice to have but not vital) 

This way we can then fully complete the process and not have to worry about coming back for licenses at a later date

**Comment by SmoothRob** [comment]
Very interested in this - We also migrate the off-boarded users OneDrive into a SharePoint Archive.

**Comment by mds-uk** [comment]
Came here to see if this had been requested! We also offload to SharePoint to save on licensing costs. The option to move data to a SharePoint library would be fantastic.
